config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
 
+  devise_for :users
   get "/articles/index"
```

Comment:
remove this route

Reply:
devise_for :users

If I remove this, tests are failing, It removed all of user routes